### ISSUE TITLE
Province for US/CA data no longer loading

### ISSUE THREAD

**Issue by mgatchis** [IYCI/covid-19-map#13]
First off... Beautiful app!

I first loaded it last night and it was working flawlessly. For some reason, now the Province/State data no longer loads for CA/US. However, China still works fine. - I thought it might have been some data inconsistencies in the API that got updated but everything looks fine as far as I can tell. 

I've poked around all I could but my knowledge is limited when it comes to the amount of layering you implemented. Just a heads up but would really like to learn more on how you integrated that if you get a chance to tune it up. 

Thanks for the awesome code!

**Comment by IYCI** [comment]
Yeah, the API source we use now don't supply active cases for US and Canada anymore.
Since the map use active to color each country/province by default, you don't see US and Canada after loaded. But there is data for confirmed and deaths.
I got pretty busy on other things for the past 2 weeks and didn't have time maintaining it, hopefully I can come back at this soon.
